utils/core.py

```diff
@@ -1,4 +1,4 @@
-""" Bobby B Bot common functions """
+""" Bot common functions """
```

Comment:
Probably also don't need to say Bot here, but open to leaving it if you feel strongly about it

Reply:
I just removed Bobby B without real consideration for what was left there. I have no feelings one way or another so will remove.